documentation/how-to-guides/deploy_magma_access_gateway.md

```diff
@@ -24,15 +24,20 @@ Magma Access Gateway using [snap](https://snapcraft.io/magma-access-gateway).
 - Ubuntu 20.04 LTS
   ([Ubuntu installation guide](https://help.ubuntu.com/lts/installation-guide/amd64/index.html))
 
+> **NOTE**: Unless specified otherwise, all actions in this guide are executed using `root` 
+> account.
+
 ## 1. Install
 
 > **NOTES:**<br>
 > 1. To see the list of currently supported configuration options, execute:
 > ```bash
 > magma-access-gateway.install --help
 > ```
-> 2. Unless specified otherwise, all actions in this guide are executed using `root` account.<br>
-> 3. Installation progress can be monitored at all times using `journalctl`.
+> 2. Installation progress can be monitored at all times using `journalctl`.
+> 3. Installation process requires **at least one** system restart at the end of the process. 
```

Comment:
Can you place this where this is expected (probably after the magma-access-gateway.install) and specify if the restart is done automatically or if the user has to do it. It's a bit confusing because there's already 1 automated restart.

Reply:
Please see my comment above regarding moving the note. 
As for restarts - agreed. I'll rephrase that.